comp/core/workloadmeta/collectors/util/kube_metadata_util.go

```diff
@@ -0,0 +1,40 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+//go:build kubeapiserver
+
+// Package util contains utility functions for image metadata collection
+package util
```

Comment:
I think we can choose a more specific name for this file. `kube_metadata_id` or similar.
I think we should avoid generic names like utils, types, etc. (I know we have several cases in the codebase).
Also the name of the package should match the name of the file (without the `_`).